homeassistant/helpers/event_decorators.py

```diff
@@ -0,0 +1,81 @@
+""" Event Decorators for custom components """
+
+import functools
+
+from homeassistant.helpers import event
+
+HASS = None
+
+
+def _callback(action, *args, **kwargs):
```

Comment:
Can you add `HASS` as the first argument to this method? That way we could potentially clean up the reference to HASS after bootstrap.